planty/users/forms.py

```diff
@@ -0,0 +1,18 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+
+
+class UserRegisterForm(UserCreationForm):
+    email = forms.EmailField()
+
+    class Meta:
+        model = User
+        fields = ['username', 'email', 'password1', 'password2']
+
+    # Hide help text in the form
```

Comment:
I would move this comment above the for loop where you actually change `help_text`